app/models/helper.py

```diff
@@ -0,0 +1,53 @@
+from flask import request
+from app.models.video import Video
+from app.models.rental import Rental
+from app.models.customer import Customer
+
+def invalid_cust_data(request_body):
+    if "name" not in request_body:
+        valid = {"details": "Request body must include name."}
+    elif "phone" not in request_body:
+        valid = {"details": "Request body must include phone."}
+    elif "postal_code" not in request_body:
+        valid = {"details": "Request body must include postal_code."}
+    else:
+        valid = False
+    return valid
+    
+def invalid_customer(customer_id):
+    invalid = False
+    if not customer_id.isnumeric():
+        invalid = {"message": f"Invalid customer id"}, 400
+    elif customer_id.isnumeric():
+        if Customer.query.get(customer_id) is None:
+            invalid = {"message": f"Customer {customer_id} was not found"}, 404
+    return invalid
+
+def invalid_video_data(request_body):
+    if "title" not in request_body:
+        valid = {"details": "Request body must include title."}
+    elif "release_date" not in request_body:
+        valid = {"details": "Request body must include release_date."}
+    elif "total_inventory" not in request_body:
+        valid = {"details": "Request body must include total_inventory."}
+    else:
+        valid = False
+    return valid
+
+def invalid_video(video_id):
```

Comment:
This might be better called something like `get_validated_video`. It would be great if instead of only returning the boolean on success, it returns the actual video. In that case, we don't have such an easy way to distinguish between successful and unsuccessful results. In that case, we might return a tuple of values, where the first part might be a boolean representing valid or invalid, and the second part might hold the error dictionary on failure, or the actual video record on success.

So on success: `return True, looked_up_video`
and on failure: `return False, error_message`

Now for the case of type consistency (as mentioned above, we might pack this all into a single dictionary with well-named keys (e.g., status, record, error) or make a small class to hold this information. There are lots of options, but doing so would simplify retrieving our values.

We might even consider making this method a class method of the Video class, since it's responsible for locating Video instances.